app/src/main/java/com/example/dndwiki/spellFinder/SpellFinderViewModel.kt

```diff
@@ -9,17 +9,32 @@ import com.example.dndwiki.network.FakeNetwork
 
 class SpellFinderViewModel : ViewModel() {
 
+    private val fakeNetwork = FakeNetwork()
+
     private val _spellsList = MutableLiveData<List<Spell>>()
```

Comment:
Let's also add an issue to migrate frim LiveData to Flow 

Not important yet right now, but LiveData is on its way out